tutor/src/models/grading/templates.js

```diff
@@ -29,6 +29,7 @@ class GradingTemplate extends BaseModel {
   @field default_due_time = '21:00';
   @field default_due_date_offset_days = 7;
   @field default_close_date_offset_days = 7;
+  @field has_task_plans;
```

Comment:
Do we want to change this here and in the representer to use `has_open_task_plans`? Can't remember if they want to allow editing up until open time.

Reply:
yeah, I was going to 🅿️ this to ask

Reply:
Yeah I think we do want to change these.